test/integration/NewNewsTest.java

```diff
@@ -0,0 +1,33 @@
+package integration;
+
+
+import models.Donation;
+import models.News;
+import models.Project;
+import models.User;
+import org.junit.Test;
+import org.openqa.selenium.By;
+
+import static org.fest.assertions.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class NewNewsTest {
+    @Test
+    public void NewNewsTestTest(){
+        DatabaseTest.runInFilledApp("testFiles/data1.yml", browser -> {
+
+            browser.goTo("http://localhost:3333/projects/2");
```

Comment:
- [ ] @Murthy10 Wenn wir ein Refactoring machen können wir ja vllt. die Magic Strings in den Tests entfernen